compiler/src/dotty/tools/dotc/typer/Applications.scala

```diff
@@ -2000,11 +2006,12 @@ trait Applications extends Compatibility {
       val strippedType2 = stripImplicit(fullType2)
 
       val result = compareWithTypes(strippedType1, strippedType2)
-      if (result != 0) result
-      else if (strippedType1 eq fullType1)
-        if (strippedType2 eq fullType2) 0         // no implicits either side: its' a draw
+      if result != 0 then result
+      else if strippedType1 eq fullType1 then
+        if strippedType2 eq fullType2
+        then drawOrOwner                          // no implicits either side: its' a draw
```

Comment:
Why do `drawOrOwner` here, as opposed to in `compareWithTypes` as you had in 39c49ad ?

We may also want to use the owner as the final tie breaker in the case where: we have an ambiguity with the strippedTypes, and that both alternatives have implicit parameters, and that the fullTypes are still ambiguous.

Reply:
It turns out there are counter-examples for both schemes. If we do `drawOrOwner` here, `StringFormatterTest` does the right thing, but slick-migrations does not work anymore. If we do `drawOrOwner` in compareTypes, slick-migratons works but `StringFormatterTest` fails. The question is what do we prioritize  case of a draw? The owner or cases without additional implicit parameters? I'll try the other scheme as well. At least StringFormatterTest has an easy fix.